src/GeneralStateTestsFiller/stCreateTest/CREATE_HighNonceMinus1DelegatecallFiller.yml

```diff
@@ -0,0 +1,75 @@
+# The test calls CREATE from an account with max allowed nonce - 1, nonce gets maximum value as a result.
+CREATE_HighNonceMinus1Delegatecall:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    c0dec0dec0dec0dec0dec0dec0dec0dec0dec0de:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          sstore(0, create(0, 0, 5))
+          sstore(1, 1)
+        }
+      nonce: '0xffffffffffffffff'
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          let type := calldataload(0)
+          if iszero(type) {
+            pop(delegatecall(sub(gas(), 1000), 0xc0dec0dec0dec0dec0dec0dec0dec0dec0dec0de, 0, 0, 0, 0))
+          }
+          if gt(type, 0) {
+            pop(callcode(sub(gas(), 1000), 0xc0dec0dec0dec0dec0dec0dec0dec0dec0dec0de, 0, 0, 0, 0, 0))
+          }
+        }
+      nonce: '0xfffffffffffffffe'
+      storage: {}
+
+  transaction:
+    data:
+      - '0x00'
+      - '0x01'
+    gasLimit:
+      - 70000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data: !!int -1
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - '>=Homestead'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
```

Comment:
sice this also covers the other case, here must be expect: shouldnotxist the account that would have been created if the nonce rule worked. (you can calculate it for create2) 
check the nonce value of accounts b94, c0dec
I think the expect section better be separate

Reply:
Added appropriate 'shouldnotexist' tests in latest push.